File: Problems/AreaOfARectangularRoom_7/AreaOfARectangularRoom.py
factor = 0.09290304

while True:
  length = input("What is the length of the room in feet? ")
  width = input("What is the width of the room in feet? ")
  if length.isnumeric() and width.isnumeric(): 
    break
  else:
    print("Please enter two positive numbers")
    
lengthFloat = float(length)
widthFloat = float(width)

feetArea = lengthFloat * widthFloat;
metersArea = feetArea * factor;

print("You entered dimensions of %f feet by %f feet." % (lengthFloat, widthFloat))
print("The area is\n %f square feet\n %f square meters\n" % (feetArea, metersArea))

File: Problems/AreaOfARectangularRoom_7/AreaOfARectangularRoom.cpp
#include <iostream>
#include <string>
#include <stdlib.h>

using namespace std;

int main()
{
	const double factor = 0.09290304;
	
	string length, width;
	double feetArea, metersArea, lengthFloat, widthFloat;
	
	cout << "What is the length of the room in feet?";
	cin >> length;
	cout << "What is the width of the room in feet?";
	cin >> width;
	
	lengthFloat = atof(length.c_str());
	widthFloat = atof(width.c_str());
	
	feetArea = lengthFloat * widthFloat;
	metersArea = feetArea * factor;
	
	cout << "You entered dimensions of " << lengthFloat << " feet by " << widthFloat << " feet.";
	cout << "The area is\n" << feetArea << " square feet\n" << metersArea << " square meters\n";
} 

/*TO DO: Check if input is valid*/
